crates/ruma-client-api/src/r0/room/create_room.rs

```diff
@@ -129,27 +129,31 @@ pub struct CreationContent {
     /// A reference to the room this room replaces, if the previous room was upgraded.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub predecessor: Option<PreviousRoom>,
+
+    /// The room type that shall be set. This is currently used for spaces
```

Comment:
```suggestion
    /// The room type.
    ///
    /// This is currently only used for spaces.
```

Reply:
Manually applied in 5fac97ae0653260c307b7779259fc95cb6f08d75